### PR TITLE
Fixed whitespaced StdErr issue

### PR DESCRIPTION
**Pull request by t-dedah**
Temporary fix for this https://github.com/Azure/arm-deploy/issues/48

### CODE REVIEW

**Comment by t-dedah** [comment]
Merging this PR as it's affecting everyone. Will revert these changes after azure-cli releases a fix for the issue.

File: src/deploy/scope_managementgroup.ts [hunk 1]
```diff
@@ -26,17 +26,18 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
...
     let commandOutput = '';
+    let commandStdErr = '';
     const deployOptions: ExecOptions = {
         silent: true,
         ignoreReturnCode: true,
-        failOnStdErr: true,
+        failOnStdErr: false,
         listeners: {
             stderr: (data: BufferSource) => {
-                core.error(data.toString());
```
Comment (bishal-pdMSFT) [inline src/deploy/scope_managementgroup.ts:35]: @t-dedah can we not achieve the same fix by trim and check for length here?

File: src/deploy/scope_managementgroup.ts [hunk 2]
```diff
@@ -26,17 +26,18 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
 
     // configure exec to write the json output to a buffer
     let commandOutput = '';
+    let commandStdErr = '';
     const deployOptions: ExecOptions = {
         silent: true,
         ignoreReturnCode: true,
-        failOnStdErr: true,
+        failOnStdErr: false,
```
Comment (bishal-pdMSFT) [inline src/deploy/scope_managementgroup.ts:33]: is this change really needed if we do the below empty check?

Comment (bishal-pdMSFT) [reply src/deploy/scope_managementgroup.ts:33]: How is this different from throwing error in the listener?
